ui/apps/platform/src/Containers/Audit/ListeningEndpoints/ListeningEndpointsPage.tsx

```diff
@@ -68,10 +79,14 @@ export function getRequestQueryStringForAutocomplete(
     //      filter in the query string
     delete filter[category];
 
-    return Object.entries(filter)
-        .map(([key, val]) => `${key}:${Array.isArray(val) ? val.join(',') : (val ?? '')}`)
-        .concat(`${category}:${value}`)
-        .join('+');
+    const contextQueryString = getRequestQueryStringForSearchFilter(
+        applyRegexSearchModifiers(filter)
+    );
+    const autocompleteSearchString = `${category}:${value ? `r/${value}` : ''}`;
+
+    return contextQueryString
+        ? `${contextQueryString}+${autocompleteSearchString}`
+        : autocompleteSearchString;
```

Comment:
Slight functionality adjustment so that autocomplete handles partial matches like other search instances in the app.